packages/video_player/video_player_avfoundation/darwin/video_player_avfoundation/Sources/video_player_avfoundation/FVPVideoPlayerPlugin.m

```diff
@@ -82,6 +82,8 @@ @interface FVPVideoPlayer ()
 @property(nonatomic) CGAffineTransform preferredTransform;
 @property(nonatomic, readonly) BOOL disposed;
 @property(nonatomic, readonly) BOOL isPlaying;
+// Playback speed when video is playing.
+@property(nonatomic, readonly) NSNumber *playbackSpeed;
```

Comment:
It would be more accurate to call this `targetPlaybackSpeed` (and document it as something like "The target playback speed requested by the plugin client."), since it's entirely possible for the playback speed to *not* be this value while playing.